src/filemap.rs

```diff
@@ -85,21 +85,38 @@ pub fn write_file(text: &StringBuffer,
         }
     }
 
+    fn diff(text: &StringBuffer,
+            filename: &str,
+            config: &Config)
+            -> Result<Vec<Mismatch>, io::Error> {
+        let mut f = try!(File::open(filename));
+        let mut ori_text = String::new();
+        try!(f.read_to_string(&mut ori_text));
+        let mut v = Vec::new();
+        try!(write_system_newlines(&mut v, text, config));
```

Comment:
check the line below which is the one that was generating the diff as it has been changed to a return